File: sdk/java/src/main/java/com/kucoin/universal/sdk/generate/service/UTAServiceImpl.java

package com.kucoin.universal.sdk.generate.service;

import com.kucoin.universal.sdk.generate.uta.account.AccountApi;
import com.kucoin.universal.sdk.generate.uta.account.AccountApiImpl;
import com.kucoin.universal.sdk.generate.uta.affiliate.AffiliateApi;
import com.kucoin.universal.sdk.generate.uta.affiliate.AffiliateApiImpl;
import com.kucoin.universal.sdk.generate.uta.market.MarketApi;
import com.kucoin.universal.sdk.generate.uta.market.MarketApiImpl;
import com.kucoin.universal.sdk.generate.uta.order.OrderApi;
import com.kucoin.universal.sdk.generate.uta.order.OrderApiImpl;
import com.kucoin.universal.sdk.generate.uta.positions.PositionsApi;
import com.kucoin.universal.sdk.generate.uta.positions.PositionsApiImpl;
import com.kucoin.universal.sdk.generate.uta.viplending.VIPLendingApi;
import com.kucoin.universal.sdk.generate.uta.viplending.VIPLendingApiImpl;
import com.kucoin.universal.sdk.internal.interfaces.Transport;


public class UTAServiceImpl implements UTAService{
    private AccountApi account;
    private MarketApi market;
    private VIPLendingApi vipLending;
    private PositionsApi positions;
    private OrderApi order;
    private AffiliateApi affiliate;

    public UTAServiceImpl(Transport transport) {
        this.account = new AccountApiImpl(transport);
        this.market = new MarketApiImpl(transport);
        this.vipLending = new VIPLendingApiImpl(transport);
        this.positions = new PositionsApiImpl(transport);
        this.order = new OrderApiImpl(transport);
        this.affiliate = new AffiliateApiImpl(transport);
    }


    public AccountApi getAccountApi() {
        return this.account;
    }

    public MarketApi getMarketApi() {
        return this.market;
    }

    public VIPLendingApi getVIPLendingApi() {
        return this.vipLending;
    }

    public PositionsApi getPositionsApi() {
        return this.positions;
    }

    public OrderApi getOrderApi() {
        return this.order;
    }

    public AffiliateApi getAffiliateApi() {
        return this.affiliate;
    }
}

File: sdk/php/src/Generate/Service/UTAServiceImpl.php
<?php
namespace KuCoin\UniversalSDK\Generate\Service;

use KuCoin\UniversalSDK\Generate\Uta\Account\AccountApi;
use KuCoin\UniversalSDK\Generate\Uta\Account\AccountApiImpl;
use KuCoin\UniversalSDK\Generate\Uta\Affiliate\AffiliateApi;
use KuCoin\UniversalSDK\Generate\Uta\Affiliate\AffiliateApiImpl;
use KuCoin\UniversalSDK\Generate\Uta\Market\MarketApi;
use KuCoin\UniversalSDK\Generate\Uta\Market\MarketApiImpl;
use KuCoin\UniversalSDK\Generate\Uta\Order\OrderApi;
use KuCoin\UniversalSDK\Generate\Uta\Order\OrderApiImpl;
use KuCoin\UniversalSDK\Generate\Uta\Positions\PositionsApi;
use KuCoin\UniversalSDK\Generate\Uta\Positions\PositionsApiImpl;
use KuCoin\UniversalSDK\Generate\Uta\VIPLending\VIPLendingApi;
use KuCoin\UniversalSDK\Generate\Uta\VIPLending\VIPLendingApiImpl;

class UTAServiceImpl implements UTAService
{
    private $transport;
    private $account;
    private $affiliate;
    private $market;
    private $order;
    private $positions;
    private $vipLending;

    public function __construct($transport)
    {
        $this->transport = $transport;
        $this->account = new AccountApiImpl($transport);
        $this->affiliate = new AffiliateApiImpl($transport);
        $this->market = new MarketApiImpl($transport);
        $this->order = new OrderApiImpl($transport);
        $this->positions = new PositionsApiImpl($transport);
        $this->vipLending = new VIPLendingApiImpl($transport);
    }

    public function getAccountApi(): AccountApi
    {
        return $this->account;
    }

    public function getAffiliateApi(): AffiliateApi
    {
        return $this->affiliate;
    }

    public function getMarketApi(): MarketApi
    {
        return $this->market;
    }

    public function getOrderApi(): OrderApi
    {
        return $this->order;
    }

    public function getPositionsApi(): PositionsApi
    {
        return $this->positions;
    }

    public function getVIPLendingApi(): VIPLendingApi
    {
        return $this->vipLending;
    }
}
